splunk_connect_for_snmp_poller/manager/mib_server_client.py

```diff
@@ -58,38 +56,24 @@ def get_translation(var_binds, mib_server_url, data_format):
     # Send the POST request to mib server
     headers = {"Content-type": "application/json"}
     endpoint = "translation"
-    TRANSLATION_URL = os.path.join(mib_server_url.strip("/"), endpoint)
-    logger.debug(f"[-] TRANSLATION_URL: {TRANSLATION_URL}")
-
-    # Set up the request params
-    params = {"data_format": data_format}
+    translation_url = os.path.join(mib_server_url.strip("/"), endpoint)
+    logger.debug(f"[-] TRANSLATION_URL: {translation_url}")
 
     try:
         # use Session with Retry
-        retry_strategy = Retry(
-            total=3,
-            backoff_factor=1,
-            status_forcelist=[429, 500, 502, 503, 504],
-            method_whitelist=["GET", "POST"],
-        )
-        adapter = HTTPAdapter(max_retries=retry_strategy)
-        session = requests.Session()
-        session.mount("https://", adapter)
-        session.mount("http://", adapter)
-        resp = session.post(
-            TRANSLATION_URL, headers=headers, data=payload, params=params, timeout=60
-        )
-
+        async with ClientSession() as session:
```

Comment:
why do we need 
_async with ClientSession() as session_ if in trap we do the same with just session = ClientSession?

Reply:
It is Python equivalent of autoclosable feature. I missed it in trap. I'll add it now.

Reply:
and same question as in trap, was removal of the retries a wanted change in logic?